### PR TITLE
Fix length and volume for wood materials, rename stick to stout branch

### PR DESCRIPTION
**Pull request by wapcaplet**
#### Summary

SUMMARY: Content "Fix length and volume for wood materials, rename stick to stout branch"

#### Purpose of change

To give the 2x4, stick, wood sheet and other items a sensible length

#### Describe the solution

Defines "longest_side" for items in `data/json/items/resources/wood.json` with reasons:

- 2x4 plank: Existing 4400 ml with 1.5"x3.5" actual cross-section yields ~51" = 130 cm length
- log: Can be cut into potentially several 2x4s, so made slightly longer at 140 cm
- stout branch (formerly "heavy stick"): As long as a 2x4 but thinner
- long stout branch (formerly "long stick"): Long enough to break into 2 stout branches
- splintered wood: Only  has 1/5 the volume of the stout branch, so I assumed it's a slender 30 cm (about 1 foot, and just over an inch in diameter)
- wood panel: Assume standard 4x4 feet x 1/2" = total volume 19 L
- wood sheet: Assume standard 4x8 feet x 1/2" = total volume 38 L
- 10-foot pole, 8-foot beam: gimmes

The volumes and weights of several of these were also tweaked to yield more realistic wood densities; now all the wood items in this file have a density around 0.60 g/cm^3, a typical density for [many different kinds of wood](https://www.engineeringclicks.com/density-of-wood/).

Here is a [Google Docs spreadsheet](https://docs.google.com/spreadsheets/d/1rmzkJMLFA8d2AuIe9Ud95klc8L1hmhuKnJ196Ity5Kk/edit?usp=sharing) that I used to work out the dimensions and density for each item.

image

#### Describe alternatives you've considered

Letting the "long stick" remain ironically named at 5 inches.

I wanted to make the splintered wood longer (around 60 cm) but this made it too long to fit in most existing backpacks, so I reduced it to 30 cm (~1 foot) to make it possible to carry a backpack full of firewood kindling.

#### Testing

Ran game, checked items in debug menu. Chopped logs into planks and splintered wood. See that everything is too long to fit in my backpack.

#### Additional context

The existing volumes for the wood panel and wood sheet worked out to about 2 mm thickness, so changing those seemed essential. Mostly my objective here was to define the lengths.

### CODE REVIEW

File: data/json/items/resources/wood.json [hunk 1]
```diff
@@ -37,13 +39,14 @@
     "id": "stick",
     "symbol": "/",
     "color": "brown",
-    "name": { "str": "heavy stick" },
-    "description": "A sturdy, heavy stick.  Makes a decent melee weapon.",
+    "name": { "str": "stout branch" },
```
Comment (kevingranade) [inline data/json/items/resources/wood.json:42]:
```suggestion
    "name": { "str": "stout branch", "str_pl": "stout branches" },
```

File: data/json/items/resources/wood.json [hunk 2]
```diff
@@ -53,13 +56,14 @@
     "id": "stick_long",
     "symbol": "/",
     "color": "brown",
-    "name": { "str": "long stick" },
-    "description": "A long stick.  Makes a decent melee weapon, and can be broken into heavy sticks for crafting.",
+    "name": { "str": "long stout branch" },
```
Comment (kevingranade) [inline data/json/items/resources/wood.json:59]:
```suggestion
    "name": { "str": "long stout branch", "str_pl": "long stout branches" },
```

**Comment by Xenomorph-III** [comment]
Question: what happens to the name of the "long pole"?

**Comment by wapcaplet** [comment]
> Question: what happens to the name of the "long pole"?

The name seems OK - do you have reason in mind to change it? Maybe it could be a "ten-foot pole" just for fun.

But now that you mention it, I see a curiosity in the weight/volume of the long stout branch versus the long pole:

- Long stout branch: 3500 g, 2500 ml
- Long pole: 2250 g, 3750 ml

Their densities are very different - the long stout branch (formerly long stick) is 1.4 g/ml, considerably denser than oak or even ebony, while the long pole is only 0.6 g/ml, closer to the density of elm or maple.[1](https://cedarstripkayak.wordpress.com/lumber-selection/162-2/)

For whatever reason, the "long pole" also gets a slightly higher bashing value (19 vs. 18), and has a bunch of polearm and reach-attack flags that the long stick/stout branch gets none of. I'd interpret all this to mean the "long pole" is a fabricated tool (it can be crafted from the wood beam, somehow, despite the wood beam only being 8 feet long), with a smooth shaft that makes it more practical as a weapon than the "long stout branch", which is just a rough unfinished piece of tree branch.

I'm inclined to at least adjust the densities of these two items. The long pole is probably OK, but 1.4 g/ml is far too dense for a typical tree branch, so I may increase the volume to bring the density around 0.6.

File: data/json/items/resources/wood.json [hunk 3]
```diff
@@ -53,13 +56,14 @@
     "id": "stick_long",
     "symbol": "/",
     "color": "brown",
-    "name": { "str": "long stick" },
-    "description": "A long stick.  Makes a decent melee weapon, and can be broken into heavy sticks for crafting.",
+    "name": { "str": "long stout branch", "str_pl": "long stout branches" },
+    "description": "A straight section of wood from a tree branch, about eight feet long and a couple inches in diameter.  Makes a decent melee weapon, and can be broken into shorter pieces for crafting.",
```
Comment (chaohedgehogs) [inline data/json/items/resources/wood.json:60]:
```suggestion
    "description": "A straight section of wood from a tree branch, about eight feet long and a couple of inches in diameter.  Makes a decent melee weapon, and can be broken into shorter pieces for crafting.",
```

**Pull request merged**